pyproject.toml

```diff
@@ -186,9 +183,6 @@ ignore = [
     "S324",   # hashlib-insecure-hash-function
     "S403",   # suspicious-pickle-import
     "S404",   # suspicious-subprocess-import
-    "A002",   # builtin-argument-shadowing
-    "C416",   # unnecessary-comprehension
-    "PT018",  # pytest-composite-assertion
```

Comment:
Are we going to lint also the test scripts?

Reply:
> Are we going to lint also the test scripts?

I'm agnostic to that; historically, I'm less worried about tests being as "lint free" as the other code, but ultimately I think we need to make that call as a group.